### ISSUE TITLE
Bug: Offline time does not coincide with the set timezone.

### ISSUE THREAD

**Issue by JourneyOver** [Monitorr/Monitorr#165]
## Your environment:

- **Operating system**: Windows 10 Pro x64 build: 1803

- **Webserver**:       NGINX v. 1.14.0

- **PHP Version**:      7.2.5 x64 NTS

- **Monitorr Version**:   1.6.2d  (in the footer of the Monitorr UI)

## Describe your issue:

I have read the t-shooting Wiki at: https://github.com/Monitorr/Monitorr/wiki/06-Troubleshooting

Pretty much as the title states and what I have said in discord, whenever a service goes offline the offline banner does not coincide with server time. 

Example: A service is up one moment and then it goes down but monitorrs offline banner time shows that it went offline hours ago even though it only went offline a minute ago.

I have timezone and everything else set correctly.

**Comment by seanvree** [comment]
@JourneyOver  Got it.  Did you set your server timezone different than what timezone it's actually in?

**Comment by JourneyOver** [comment]
@seanvree nope, I'm running everything off of the computer that I am currently using and timezone and all that are correct on the computer, along with the time that shows up on the monitorr clock, it seems to only be the offline banner time that ends up being wrong.

**Comment by seanvree** [comment]
@JourneyOver  got it.  So just to be clear, when it FIRST goes offline is the offline time in the banner right, or it's off no matter what?

**Comment by seanvree** [comment]
@JourneyOver  humm, weird.  Can you do me a favor?  Can you download the file at the location below:

https://gist.github.com/seanvree/48fff2655245a4b3b3cd330f5155bdcf

And use the downloaded file to replace the file at /assets/php/check.php  ?

just rename your current check.php to check.php.old and put this file in it's place.

After you do that, make sure all your services are online, then bring one or two offline.  You should now see a timezone after the actual time in the offline banner.  Is that timezone the same timezone as the clock in the UI?

**Comment by JourneyOver** [comment]
done that and can pretty much say nope not at all on the banner and the timezone showing the same, The timezone that shows for the banner is coming up as just UTC, while my server and the settings I have monitorr set for are set as both central time.

**Comment by seanvree** [comment]
Okay, I know what's up.  Two more questions... is the actual server in central time?

Also,  do you know where your php.ini file is? Can you check the value at approx line 950, what is this value?

```
[Date]
; Defines the default timezone used by the date functions
; http://php.net/date.timezone
date.timezone = "US/Pacific"
```

![image](https://user-images.githubusercontent.com/8906637/40871796-75c36790-65f7-11e8-9c62-161ec4b9b929.png)

**Comment by JourneyOver** [comment]
Yes actual server/computer is in central time as I am currently in central time typing on said server/computer and the time on the clock is correct xD

as for the php.ini file date.timezone, that section seems to be commented out. I used https://github.com/elmerfdz/OrganizrInstaller ( specifically this https://github.com/elmerfdz/OrganizrInstaller/tree/master/windows/o2wi ) to initially set up nginx and all that and it looks like it default came that way https://github.com/elmerfdz/OrganizrInstaller/blob/master/windows/o2wi/config/php.ini#L939 and I haven't touched the php.ini file any yet.

**Comment by seanvree** [comment]
Yeah, cool.  It's actually a bug in our code, but yeah, that just tells any php script that is running on your webserver to use that timezone.  So, it's good to set that, but I'll make a change in our code. 

Thanks brother. 

Cheers.

**Comment by JourneyOver** [comment]
Not a problem at all :) as said glad we figured it out and found an issue in monitorrs code in the process.

**Comment by seanvree** [comment]
TO DO:  Add global timezone variable in check.php from $jsonusers['timezone'];

```
        $timezone = $jsonusers['timezone'];

        date_default_timezone_set($timezone);
```

Will push with next version.

**Comment by seanvree** [comment]
FIXED: with v 1.7.6 https://github.com/Monitorr/Monitorr/releases
